changelog.json

```
@@ -1,5 +1,17 @@
 {
   "releases": {
+    "2.1.0-beta1": [
+      "[New] Commit form will prompt user when working on a protected branch - #7826",
```

I'm wondering if warn is more accurate than prompt, given that we use the warning icon?
```suggestion
      "[New] Commit form will warn user when working on a protected branch - #7826",
```

---

I dig it